### ISSUE TITLE
Metodo PedroTeixeira_Correios_Model_Carrier_CorreiosMethod()

### ISSUE THREAD

**Issue by meganovidades** [pedro-teixeira/correios#186]
Pessoal eu tinha uma mini pagina aqui com esta chamada para fazer algumas atualizações de status conforme o status dos correios, mas ao atualizar o modulo parou de funcionar, abaixo segue o trecho que eu utlizava, alguem poderia me ajudar de como funcionar novamente? Obrigado!

foreach($trackinList as $tracking)
			{
				$correiosMethod = new PedroTeixeira_Correios_Model_Carrier_CorreiosMethod();
				$r              = $correiosMethod->getTracking($tracking['track_number'])->getAllTrackings();
				if(count($r[0]) > 0)
				{
					$status    = $r[0]['status'];
					$statusfim = $r[0]['activity'];
Aqui ele pegava o retorno das variaveis acima, porém esta voltando vazia agora.

o que devo mudar?
este metodo não funciona mais?

**Comment by rafaelpatro** [comment]
Sim. O método _getTracking_ foi totalmente modificado neste commit: 0fa436547b480acf03c6ee898e2c922d55c9dc9b.
Sugiro exibir a variável _$r_ para entender melhor como extrair o resultado.
